nerfstudio/utils/colormaps.py

```diff
@@ -80,6 +80,9 @@ def apply_colormap(
     if image.dtype == torch.bool:
         return apply_boolean_colormap(image)
 
+    if image.shape[-1] > 3:
```

Comment:
Can you update the docstring to inform users that PCA is used when >3 channels (line 51)